src/style.scss

```diff
@@ -455,12 +455,10 @@ input {
 
 :host([validity]:not([validity='valid'])) {
   .inputElement {
-    &:invalid {
-      border-color: var(--auro-color-alert-error-on-light);
+    border-color: var(--auro-color-alert-error-on-light);
 
-      @include auro_breakpoint--sm {
-        outline-color: var(--auro-color-alert-error-on-light);
-      }
+    @include auro_breakpoint--sm {
+      outline-color: var(--auro-color-alert-error-on-light);
```

Comment:
These changes are resulting in a 1px border rather than a 2px  border on the error state.

Reply:
When running the demo off of `main`, auro-input's border in error state is still 1px thick. This must be a different bug.